[decrypt_all] Check if file needs to decrypted or not for speed up large oc setups.

chriseqipe opened the pull request.

nickvergessen committed:
Fix Share::getUsersSharingFile() for mount points

DeepDiver1975 committed:
Initial implementation of the new chunked upload - as specified in https://dragotin.wordpress.com/2015/06/22/owncloud-chunking-ng/

chriseqipe committed:
Check if file needs to decrypted or not for speed up large oc setups.

LukasReschke commented on lib/private/encryption/decryptall.php:
```diff
@@ -213,9 +214,14 @@ protected function decryptUsersFiles($uid, ProgressBar $progress, $userCount) {
 						$progress->setMessage("decrypt files for user $userCount: $path");
 						$progress->advance();
-						if ($this->decryptFile($path) === false) {
+						if ($file->getData()['encrypted'] == 0) {
```
Please always use `===`. If you want to typecast here use `(int)$file->getData()['encrypted']`.

chriseqipe committed:
Respect oc coding style guide.

chriseqipe commented:
Thx for the hint. In fact $file->getData()['encrypted'] comes as "0" String. I decided to cast it to int.

rullzer committed:
Add lazyRootFolder

rullzer committed:
Make shareproviders use the lazy root folder

rullzer committed:
Add intergration tests for new chunking

The same intergration tests as for the old endpoint.
But now using the new chunking.

We upload 3 chunks in different order and the result should be the same
in all three.

PVince81 commented:
@schiesbn what do you think ?

MorrisJobke commented:
@LukasReschke @schiesbn Please review ;)

rullzer committed:
Set proper permissions on link share

If we do not allow public upload we should limit the permissions on
links shares upon retrieval.

* Added unit test
* Allow fetching federated shares by token as well

rullzer committed:
Move public webdav auth over to share manager

The public webdav auth should use the shiny new share manager.

rullzer committed:
Add unit tests for public auth webdav

RobinMcCorkell committed:
Fix conditional check in MySQL setup

Commit:
re-try to send unshare notification if remote server is not available

nickvergessen committed:
Update 3rdparty with sabre/dav 3.0.9

stweil committed:
lib: Fix typos (found by codespell)

Signed-off-by: Stefan Weil <sw@weilnetz.de>

rullzer committed:
Remove pre/post_addToGroup hooks for shares

There is no need to perform the checks for unique targets on add to
group as we have to do this all when mounting the shares anyway.

rullzer committed:
Fix Share20OCSTest

rullzer committed:
Fix ShareManager tests

rullzer committed:
Fix avatar controller tests

rullzer committed:
Fix notification tests

rullzer committed:
Fix mappertest

nickvergessen committed:
Correctly return an empty string for empty files

DeepDiver1975 committed:
Merge pull request #23848 from stweil/master

lib: Fix typos (found by codespell)

DeepDiver1975 committed:
Merge pull request #23736 from owncloud/move_public_webdav_auth_to_share_manager

Move public webdav auth to share manager

DeepDiver1975 committed:
Merge pull request #23553 from owncloud/lock-ttl-execution-time

Make lock ttl configurable

DeepDiver1975 committed:
Merge pull request #23839 from owncloud/sabre-dav-3.0.9

Update 3rdparty with 3.0.9 sabre/dav

rullzer committed:
Fix filesplugin

rullzer committed:
Fix encryption tests

rullzer committed:
Fix dav comments

DeepDiver1975 committed:
Merge pull request #23510 from owncloud/birthdays-on-shared-addressbooks

Propagate birthdays of shared addressbooks to the sharee's birthday c…

scrutinizer-auto-fixer committed:
Scrutinizer Auto-Fixes

This commit consists of patches automatically generated for this project on https://scrutinizer-ci.com

Commit:
[tx-robot] updated from transifex

Commit:
[tx-robot] updated from transifex

Commit:
Added white download icons

DeepDiver1975 committed:
Adding VCFExportPlugin

DeepDiver1975 committed:
Merge pull request #23893 from owncloud/add-VCFExportPlugin

Adding VCFExportPlugin

DeepDiver1975 committed:
Merge pull request #23891 from owncloud/add-download-icon-white

Added white download icons

DeepDiver1975 committed:
Merge pull request #23863 from owncloud/fix_unit_tests

Fix unit tests warnings

Commit:
[tx-robot] updated from transifex

DeepDiver1975 committed:
Merge pull request #23860 from owncloud/followup-23304-opening-empty-file

Correctly return an empty string for empty files

DeepDiver1975 committed:
Merge pull request #22615 from owncloud/add-blacklisted-files-to-capabilities

Add blacklisted_files to capabilities

DeepDiver1975 committed:
Fixes #23899

DeepDiver1975 committed:
Merge pull request #23868 from owncloud/scrutinizer-patch-2

Scrutinizer Auto-Fixes

MorrisJobke committed:
Merge pull request #23798 from owncloud/federated-unshare-background-job

re-try to send unshare notification if remote server is not available

LukasReschke committed:
Move login form into controller

First step on getting the authorisation stuff cleaned up. This is only for the login form, all other stuff is still where it is.

DeepDiver1975 committed:
Move version check code out of class Updater

josh4trunks committed:
To be safe don't unload until end (#23752)

* To be safe don't unload until end

* unlink needs to be before return to actually do anything

MorrisJobke committed:
Catch the AutoloadNotAllowedException also for legacy jobs (#23901)

* same as #18839 for legacy jobs
* avoids spamming the log with useless entries

nickvergessen committed:
Always add activities for comments to the stream

nickvergessen committed:
Merge pull request #23856 from owncloud/share_remove_addtogroup_hooks

Remove pre/post_addToGroup hooks for shares

ChristophWurst committed:
Show tooltip if the app is downloaded from a remote server
fixes #14405

DeepDiver1975 committed:
Suggest cli based updater in case the instance is bigger - #23913

Commit:
[tx-robot] updated from transifex

rullzer committed:
Don't calculate permissions twice (#23917)

There is no need to calculate the permissions twice.
Saves some is_dir and file_exists calls.

rullzer committed:
Migrate post_groupDelete hook to share manager (#23841)

The hook now calls the share manager that will call the responsible
shareProvider to do the proper cleanup.

* Unit tests added

Again nothing should change it is just to cleanup old code

Henni committed:
Checkboxes (#23063)

* remove ie8 fixes for checkboxes

* add indeterminate style to checkboxes

* add styles for radio inputs

* add missing checkbox-mixed-disabled icon

* fix white radios

* remove unnecessary png file

ChristophWurst committed:
load file sorting mode from the db

ChristophWurst committed:
persist file sorting changes

georgehrke committed:
update 3rdparty to latest JSqueeze

ChristophWurst committed:
fix default value, update js/php tests

LukasReschke committed:
Exclude autoload_static.php (#23935)

Composer 1.1 has since yesterday a new performance improvement that will be automatically used for PHP >= 5.6, however this file is incompatible with older PHP versions and thus we need to exclude it from the checks.

Note that this performance improvement is only used on >= 5.6 so ownCloud will still run fine on older PHP versions as well.

DeepDiver1975 committed:
Add unit testing capabilities for templates (#23708)

Add unit testing capabilities for templates

stweil committed:
apps: Fix typos (found by codespell) (#23862)

Signed-off-by: Stefan Weil <sw@weilnetz.de>

LukasReschke committed:
Employ a stricter Content Security Policy on remote.php

Items sent by remote.php have not to be interpreted by browsers in any way.

DeepDiver1975 committed:
Fix chunk file move

DeepDiver1975 committed:
Document an example wait timeout for MySQL in case the web hoster is killing the connection too early - fixes #15407

DeepDiver1975 committed:
Revert "Document an example wait timeout for MySQL in case the web hoster is killing the connection too early - fixes #15407"

This reverts commit acad7b30bef8ec11f6e57ee7bfcafec4b8795e03.

DeepDiver1975 committed:
Document an example wait timeout for MySQL in case the web hoster is killing the connection too early - fixes #15407

This reverts commit e9a9af3493caa7e839e18cda0c8ea2d8dd7bfd22.

Commit:
always return the complete permissions the file was shared with

DeepDiver1975 committed:
Fix unit test of file plugin

icewind1991 committed:
chunk the cleaning up of shared locks

ChristophWurst committed:
filter hidden files on the web interface

add checkbox to toggle show/hide hidden files

persist show hidden setting

fix settings menu layout

test ApiController::showHiddenFiles

don't show hidden files by default

Store config in Backbone model and inject it into FileList

Filter files only temporarily when rending the file list

Fix file rename validation

Commit:
set $share to null if getShareByToken() failed

stweil committed:
misc: Fix some typos

Some of them were found by codespell.

Signed-off-by: Stefan Weil <sw@weilnetz.de>

LukasReschke committed:
Add test

LukasReschke committed:
Merge pull request #23953 from owncloud/small_share_manager_fix

set $share to null if getShareByToken() failed

Commit:
[tx-robot] updated from transifex

MorrisJobke committed:
Use 6 months as SSL STS header threshold

* this uses 6 months (6 * 30 * 24 * 60 * 60 = 15552000)
* old value was half a year (365 / 2 * 24 * 60 * 60 = 15768000)
* fixes #23957

rullzer committed:
Let the querybuilder convert

nickvergessen committed:
Merge pull request #23945 from owncloud/fix_oc_share_permissions

always return the complete permissions the file was shared with

rullzer committed:
Move files in \OC\Activity to PSR-4

DeepDiver1975 committed:
Merge pull request #23921 from owncloud/show-app-download-tooltip

Show tooltip if the app is downloaded from a remote server

DeepDiver1975 committed:
Merge pull request #23954 from stweil/master

misc: Fix some typos

DeepDiver1975 committed:
Merge pull request #23958 from owncloud/use-6-months-as-threshold-for-ssl

Use 6 months as SSL STS header threshold

ChristophWurst committed:
restore selection after re-rendering file rows

nickvergessen committed:
Merge pull request #23963 from owncloud/lazyRoot

Add LazyRootFolder

ChristophWurst committed:
Merge pull request #23966 from owncloud/restore-file-selection

restore selection after re-rendering file rows

DeepDiver1975 committed:
Merge pull request #23962 from owncloud/activity-psr4

Move files in \OC\Activity to PSR-4

DeepDiver1975 committed:
Merge pull request #23943 from owncloud/document-mysql-wait-timeout

Document an example wait timeout for MySQL in case the web hoster is …

DeepDiver1975 committed:
Merge pull request #20118 from owncloud/chunked-upload-dav

Initial implementation of the new chunked upload

rullzer committed:
Move post_removeFromGroup to shareManager

The last sharing hook to be moved over.

* Added unit tests
* Removed old tests that relied on old behaviour
* Removed old hooks.php

icewind1991 committed:
cache the scanner/updater/cache in the wrapper storage

DeepDiver1975 committed:
Use better names for the default calendars and addressbooks - fixes #23720

DeepDiver1975 committed:
Remove dav migration for 9.1

SergioBertolinSG committed:
Added test case about downloading a file inside a folder with range

ChristophWurst committed:
extract mail view for sending share invitations
fixes #22947

rullzer committed:
Move \OC\App to PSR-4

RobinMcCorkell committed:
Fix storage backend class namespaces and move to subdir

All classes that were previously \OC\Files\Storage\FooBar are now
\OCA\Files_External\Lib\Storage\FooBar

RobinMcCorkell committed:
Fix OCA\Files\External\Api namespace

DeepDiver1975 committed:
Merge pull request #23981 from owncloud/app_psr4

Move \OC\App to PSR-4

Commit:
[tx-robot] updated from transifex

rullzer committed:
Move \OC\Lock to PSR-4

ChristophWurst committed:
Debounce heartbeat ajax calls to lower the number of requests
fixes #22397

MorrisJobke committed:
Properly handle exception within templates

* fixes partial printed templates when exception is thrown in between

rullzer committed:
Merge pull request #23986 from owncloud/psr-4_lock

Move \OC\Lock to PSR-4

MorrisJobke committed:
Merge pull request #19615 from owncloud/issue-15858-mount-points-return-wrong-paths-for-getusersharingfiles

Fix Share::getUsersSharingFile() for mount points

DeepDiver1975 committed:
Merge pull request #23976 from owncloud/kill-dav-migration

Remove dav migration for 9.1

scrutinizer-auto-fixer committed:
Scrutinizer Auto-Fixes (#23959)

* Scrutinizer Auto-Fixes

This commit consists of patches automatically generated for this project on https://scrutinizer-ci.com

* revert append() change

rullzer committed:
On mount make sure multiple shares with same target map to unique ones (#23937)

Scenario:
user0 shares a folder 'foo' with user2
user1 shares a folder 'foo' with user2
user2 logs in

Before: show only the 'foo' from user1

After: show both.

* Added intergration tests

ChristophWurst committed:
add loading feedback to user menu entries (#23916)

fixes #19857

ChristophWurst committed:
close navigation menu when opening app in new tab (#23914)

DeepDiver1975 committed:
Merge pull request #22760 from owncloud/tooltip-cssfix

add word wrap to tooltips. Fixes #22746

DeepDiver1975 committed:
Merge pull request #23975 from owncloud/change-default-calendar-and-contacts-names

Use better names for the default calendars and addressbooks

jancborchardt committed:
use background image for log in view

ChristophWurst committed:
update jquery to version 2.2.3

ChristophWurst committed:
add jquery files to git

DeepDiver1975 committed:
Translate contacts birthday - fixes #23982

DeepDiver1975 committed:
Merge pull request #23991 from owncloud/handle-exceptions-in-templates

Properly handle exception within templates

MorrisJobke committed:
Merge pull request #23985 from owncloud/ext-fix-namespaces

Fix namespaces for files_external

Commit:
Merge pull request #23527 from owncloud/fix-21136

disable Paged Search when chunksize is set to 0, fixes #21136

DeepDiver1975 committed:
Introduce isReadyForUser and verify in file transfer ownership - fixes #23786

DeepDiver1975 committed:
Kill movable 3rdparty

rullzer committed:
Move \OC\Security to PSR-4

Commit:
Merge pull request #23561 from owncloud/background-scan-unscanned

handle completely unscanned storages in the background scanner

DeepDiver1975 committed:
Merge pull request #23994 from owncloud/log-in-image

use background image for log in view

DeepDiver1975 committed:
Merge pull request #24005 from owncloud/3rdparty-has-to-be-in-root

Kill movable 3rdparty

DeepDiver1975 committed:
Merge pull request #23998 from owncloud/translate-birthday

Translate contacts birthday

DeepDiver1975 committed:
Merge pull request #24007 from owncloud/security_psr4

Move \OC\Security to PSR-4

rullzer committed:
Move \OC\Session to PSR-4

Commit:
[tx-robot] updated from transifex

DeepDiver1975 committed:
Merge pull request #24013 from owncloud/session_psr4

Move \OC\Session to PSR-4

rullzer committed:
Move SharedStorage et al over to the shareManager and jail

rullzer committed:
Fix related logic

rullzer committed:
Fix tests

rullzer committed:
Move \OC\Cache to PSR-4

rullzer committed:
Move share-permissions property namespace

Fixes https://github.com/owncloud/core/issues/23741

nickvergessen committed:
Fix deprecated usages

Warning: -convert-to is deprecated.  Use --convert-to instead.
Warning: -outdir is deprecated.  Use --outdir instead.

rullzer committed:
Merge pull request #24016 from owncloud/cache_psr4

Move \OC\Cache to PSR-4

rullzer committed:
Move \OC\Command to PSR-4

ChristophWurst committed:
downgrade jquery, update jquery-migrate

rullzer committed:
Fix resolve path

icewind1991 committed:
Fix loading the dropbox autoloader

icewind1991 committed:
Only construct the storage when we start using it

DeepDiver1975 committed:
Merge pull request #24024 from owncloud/dropbox-autoload

Fix loading the dropbox autoloader

DeepDiver1975 committed:
Merge pull request #24023 from owncloud/command_psr4

Move \OC\Command to PSR-4

DeepDiver1975 committed:
Merge pull request #23933 from owncloud/remember-file-sorting

remember file sort order

SergioBertolinSG committed:
Added test case for QA-94

rullzer committed:
Move \OC\Comments to PSR-4

DeepDiver1975 committed:
Merge pull request #24031 from owncloud/comments_psr4

Move \OC\Comments to PSR-4

LukasReschke committed:
Use !== instead of empty

Users can be named null

LukasReschke committed:
Rename `username` to `loginName`

UID and login name are two different things.

LukasReschke committed:
Keep used username in URL

This is required until the new controller can also handle POST requests

rullzer committed:
Move \OC\DB to PSR-4

Besides the statement wrapper that is moved to the legacy folder
(namepsace of shame folder)

LukasReschke committed:
Also check for an empty string

PHP. Yay.

DeepDiver1975 committed:
Merge pull request #24017 from owncloud/share-permissions-namespace

Move share-permissions property namespace

Commit:
link to file_versioning.rst

which describes files_versions and config.php settings
backports to 8.2

Commit:
[tx-robot] updated from transifex

LukasReschke committed:
Merge pull request #23939 from owncloud/stricter-csp-for-dav-backend

Employ a stricter Content Security Policy on remote.php

LukasReschke committed:
Merge pull request #24030 from owncloud/integration-tests-improved-resharing

Added test case for QA-94

LukasReschke committed:
Merge pull request #24020 from owncloud/update-sample-command

Fix deprecated usages of libre office commands

LukasReschke committed:
Merge pull request #23978 from owncloud/integration-test-remaining-from-qa-26

Added test case about downloading a file inside a folder with range

LukasReschke committed:
Merge pull request #23936 from owncloud/update_3rdparty

update 3rdparty to latest JSqueeze

Commit:
[tx-robot] updated from transifex

MorrisJobke committed:
Merge pull request #24036 from owncloud/db_psr4

Move \OC\DB to PSR-4

rullzer committed:
Move \OC\Memcache to PSR-4

DeepDiver1975 committed:
Merge pull request #24048 from owncloud/memcache_psr-4

Move \OC\Memcache to PSR-4

rullzer committed:
Move \OC\Notification to PSR-4

skjnldsv committed:
Fix rgb values

Fix was required because values was too big for rgb and breaking the brightness calculation.
Now we have the initial sat (70%) and the reduction to 60 if too bright working again.

DeepDiver1975 committed:
Merge pull request #23268 from owncloud/range-requests-should-not-block-the-session

Close session for publicwebdav

ChristophWurst committed:
Turn off jQuery animations when testing

rullzer committed:
Add data-fingerprint property to webdav

rullzer committed:
Add occ command for updating the data-fingerprint

DeepDiver1975 committed:
Merge pull request #24050 from owncloud/notification_psr4

Move \OC\Notification to PSR-4

LukasReschke committed:
Merge pull request #23947 from owncloud/login-to-ctrl

Move login form into controller

rullzer committed:
Add config text

LukasReschke committed:
Move logout to controller

Testable code. Yay.

nickvergessen committed:
Adjust notification paths to PSR-4

MorrisJobke committed:
Fix gitignore for core/vendor

* es6-promise and davclient.js

MorrisJobke committed:
remove unneeded files

MorrisJobke committed:
Update 3rdparty to remove unneeded files

DeepDiver1975 committed:
Merge pull request #24058 from owncloud/fix-notification-paths

Adjust notification paths to PSR-4

DeepDiver1975 committed:
Remove deprecated HTTPHelper

LukasReschke committed:
Properly escape URL

Fixes https://github.com/owncloud/core/issues/23499

DeepDiver1975 committed:
Merge pull request #24072 from owncloud/properly-escape-url

Properly escape URL

DeepDiver1975 committed:
Merge pull request #23922 from owncloud/upgrade-only-with-cli-for-big-installations

Suggest cli based updater in case the instance is bigger

ChristophWurst committed:
Merge pull request #23911 from owncloud/split-upgrade-command

Move version check code out of class Updater

rullzer committed:
Move copy- and moveFromStorage to jail

LukasReschke committed:
Merge pull request #24037 from owncloud/file_versions_default

link to file_versioning.rst

rullzer committed:
Move OC\IntegrityCheck to PSR-4

rullzer committed:
Fix use statement

DeepDiver1975 committed:
In case of fatal php errors and other unhandled exceptions no html error page is expected to be displayed in the console

DeepDiver1975 committed:
Fix displayname for initial calendars and address books - fixes #24057

DeepDiver1975 committed:
Revert "Fix displayname for initial calendars and address books - fixes #24057"

This reverts commit a5d3e5ed6893b35dc473a479105d95f03a403999.

DeepDiver1975 committed:
Fix displayname for initial calendars and address books - fixes #24057

DeepDiver1975 committed:
Merge pull request #24056 from owncloud/move-logout-to-logincontroller

Move logout to controller

DeepDiver1975 committed:
Merge pull request #24049 from owncloud/skjnldsv-fix-colors-values

Fix rgb values

DeepDiver1975 committed:
Merge pull request #23973 from owncloud/share_move_post_delete_from_group_hook

Move post_removeFromGroup to shareManager

Commit:
[tx-robot] updated from transifex

DeepDiver1975 committed:
Merge pull request #23903 from owncloud/fix-user-data-folder-creation-for-apache-login

Ensure the user's data folder is created for IApacheBackend authentication

DeepDiver1975 committed:
Merge pull request #24074 from owncloud/integritycheck_psr4

Move OC\IntegrityCheck to PSR-4

DeepDiver1975 committed:
Merge pull request #23993 from owncloud/update-to-jquery2

Update jquery to version 2.2.3

rullzer committed:
Move \OC\Console to PSR-4

schiesbn commented on lib/private/encryption/decryptall.php:
```diff
@@ -213,9 +214,14 @@ protected function decryptUsersFiles($uid, ProgressBar $progress, $userCount) {
 						$progress->setMessage("decrypt files for user $userCount: $path");
 						$progress->advance();
-						if ($this->decryptFile($path) === false) {
+						if ((int)$file->getData()['encrypted'] === 0) {
```
please call  `$file->isEncrypted();` instead.

Otherwise it looks good, thanks a lot for your pull request!

ChristophWurst committed:
remove deprecated jQuery.browser

ChristophWurst committed:
migrate jQuery.fn.unload()

rullzer committed:
Move moveFromCache to CacheJail

rullzer committed:
Merge pull request #24054 from owncloud/data_fingerprint

Add data-fingerprint property

LukasReschke committed:
Merge pull request #24076 from owncloud/fix-initial-calendar-and-addressbook-names

Fix displayname for initial calendars and address books

ChristophWurst committed:
replace $.parseJSON() by JSON.parse()

rullzer committed:
Move the group dummy backend to test

The dummy backend is only used for testing and should thus reside in
tests.

ChristophWurst committed:
replace $().attr('checked') by $().prop('checked', state) or $().is(':checked')

DeepDiver1975 committed:
Merge pull request #24004 from owncloud/dont-transfer-files-to-not-ready-user

Introduce isReadyForUser and verify in file transfer ownership

DeepDiver1975 committed:
Merge pull request #24079 from owncloud/console_psr4

Move \OC\Console to PSR-4

DeepDiver1975 committed:
Merge pull request #24062 from owncloud/fix-gitignore

Fix gitignore for core/vendor

Commit:
we need to initialize the mount points of the given user before we recover
access to his files

rullzer committed:
When the scanner detects a file is changed clear checksum

Fixes #23782 and #23783

If the file scanner detects a changed file we clear the checksum while
we update the cache.

* Unit test added

rullzer committed:
Move \OC\Diagnostics to PSR-4

tyrylu committed:
Fix reporting of the categories in app settings as links by screen readers.

Commit:
[tx-robot] updated from transifex

Commit:
Merge pull request #23915 from owncloud/activity-505-always-stream-for-comments

Always add activities for comments to the stream

jancborchardt committed:
Merge pull request #23980 from owncloud/remember-email-when-setting-expirationdate

remember email when setting expiration date

DeepDiver1975 committed:
Merge pull request #24103 from owncloud/diagnostics_psr4

Move \OC\Diagnostics to PSR-4

DeepDiver1975 committed:
Merge pull request #24104 from tyrylu/accessibility_settings_fix

App settings screen accessibility fixes

DeepDiver1975 committed:
Merge pull request #24097 from owncloud/fix_23632

init users mount point before recovery

DeepDiver1975 committed:
Merge pull request #24094 from owncloud/move_dummy_group_backend_to_test

Move the group dummy backend to test

DeepDiver1975 committed:
Merge pull request #23952 from owncloud/hide-hidden-files

Add option to hide hidden files

Commit:
preserve information if it is a rename operation or not

nickvergessen committed:
Add loadAdditionalScripts events to admin and personal for plugins that have no section

rullzer committed:
Move \OC\Mail to PSR-4

DeepDiver1975 committed:
Merge pull request #24120 from owncloud/mail_psr4

Move \OC\Mail to PSR-4

DeepDiver1975 committed:
Merge pull request #23760 from owncloud/fix-setup-mysql

Fix conditional check in MySQL setup

DeepDiver1975 committed:
Merge pull request #24116 from owncloud/fix_24095

preserve information if it is a rename operation or not

LukasReschke committed:
Merge pull request #24075 from owncloud/no-html-on-cli

In case of fatal php errors and other unhandled exceptions no html er…

rullzer committed:
Move \OC\Preview to PSR-4

rullzer committed:
Move \OC\Tagging to PSR-4

DeepDiver1975 committed:
With PHP7 Throwable is the new base interface for all Exception and Error - we should be capable to log Throwable as well

DeepDiver1975 committed:
Catch class Error on all root entrypoints

DeepDiver1975 committed:
In case an app has issues while loading the app is disabled and request processing continues

DeepDiver1975 committed:
Merge pull request #24123 from owncloud/preview_psr4

Move \OC\Preview to PSR-4

DeepDiver1975 committed:
Fix web updater - fixes #24115

DeepDiver1975 committed:
Merge pull request #24118 from owncloud/add-load-script-events-for-settings

Add loadAdditionalScripts events to admin and personal settings

DeepDiver1975 committed:
Merge pull request #24098 from owncloud/fix_23782

When calling file_put_contents clear the checksum

DeepDiver1975 committed:
Merge pull request #23919 from owncloud/cyclyc-share-dep-example

SharedStorage to new sharing code + cleanup

DeepDiver1975 committed:
Merge pull request #24064 from owncloud/update-3rdparty

Update 3rdparty to remove unneeded files

DeepDiver1975 committed:
Merge pull request #24081 from owncloud/migrate-deprecated-jquery-v1-functions

migrate deprecated jQuery 1.x functions

DeepDiver1975 committed:
Merge pull request #23951 from owncloud/cleanlocks-chunk

chunk the cleaning up of shared locks

MorrisJobke committed:
Merge pull request #23990 from owncloud/heartbeat-debounce

Debounce heartbeat ajax calls to lower the number of requests

DeepDiver1975 committed:
Merge pull request #24132 from owncloud/fix-web-updater

Fix web updater

DeepDiver1975 committed:
Merge pull request #24127 from owncloud/handle-app-loading-error

Handle app loading error

Commit:
[tx-robot] updated from transifex

nickvergessen committed:
Merge pull request #24133 from owncloud/tagging_psr4

Move \OC\Tagging to PSR-4

chriseqipe committed:
Merge branch 'master' of github.com:chriseqipe/core

PVince81 commented:
@chriseqipe something looks wrong with your commit list.

Can you redo a PR ? Best is to create a branch in your own repo, put the changes on the branch and submit the PR from there.

PVince81 commented:
Something went wrong with this PR, see commit list.

Please resubmit.

PVince81 commented:
Ok, I have spotted the commits, I'll make a new PR and we'll continue from there.

PVince81 commented:
I cherry picked the commits into a new PR: https://github.com/owncloud/core/pull/24628
Let's continue there
